irc/modes.go

```diff
@@ -30,6 +30,10 @@ func (client *Client) applyUserModeChanges(force bool, changes modes.ModeChanges
 		case modes.Bot, modes.Invisible, modes.WallOps, modes.UserRoleplaying, modes.Operator, modes.LocalOperator, modes.RegisteredOnly:
 			switch change.Op {
 			case modes.Add:
+				if change.Mode == modes.Operator || change.Mode == modes.LocalOperator {
+					client.server.stats.ChangeOperators(1)
```

Comment:
I believe this is redundant with #256. You might want to merge origin/master into this branch? (Or rebase, according to preference. I don't think we have a policy at this time about merge vs. rebase for PRs.)